roles/realm_join/tasks/install_pexpect.yml

```diff
@@ -0,0 +1,36 @@
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: python3-pexpect
```

Comment:
instead of having two blocks for different OSes, we could have vars like this:

https://github.com/theforeman/forklift/blob/4090e636e55d83a39d7f7cdaf5ce9b1f3ed41dc8/roles/sos_report/tasks/main.yml#L2-L8

https://github.com/theforeman/forklift/blob/4090e636e55d83a39d7f7cdaf5ce9b1f3ed41dc8/roles/sos_report/vars/Debian.yml#L2
https://github.com/theforeman/forklift/blob/4090e636e55d83a39d7f7cdaf5ce9b1f3ed41dc8/roles/sos_report/vars/RedHat.yml#L2

It's up to you if you want to go that route, for a single package it might be overkill.

Reply:
I :heart: it. Lets do it